com.unity.render-pipelines.core/Runtime/Volume/VolumeProfile.cs

```diff
@@ -30,6 +32,40 @@ void OnEnable()
             // harmless and happens because Unity does a redraw of the editor (and thus the current
             // frame) before the recompilation step.
             components.RemoveAll(x => x == null);
+
+            // Subscribe to the changes performed on the current pipeline to know which Volume Components are supported on the current pipeline
+            RenderPipelineManager.activeRenderPipelineTypeChanged += UpdateIsSupportedOnCurrentPipeline;
+            UpdateIsSupportedOnCurrentPipeline();
+        }
+
+        private void OnDisable()
+        {
+            RenderPipelineManager.activeRenderPipelineTypeChanged -= UpdateIsSupportedOnCurrentPipeline;
+        }
+
+        internal void UpdateIsSupportedOnCurrentPipeline()
+        {
+            var currentPipeline = RenderPipelineManager.currentPipeline;
+            if (currentPipeline != null)
+            {
+                var currentPipelineType = currentPipeline.GetType();
+                foreach (var component in components)
+                {
+                    SetIsSupportedOnCurrentPipeline(component, currentPipelineType);
+                }
+            }
+        }
+
+        void SetIsSupportedOnCurrentPipeline(VolumeComponent component, Type currentPipelineType)
```

Comment:
[Readability] Either rename `currentPipelineType` to `renderPipelineType` or make the method find who is the current one.

Reply:
Same goes for method name. It should be `SetIsSupportedOnPipeline` if the method don't get who is the current one